syntax_and_semantics/annotations.md

```diff
@@ -1,6 +1,6 @@
-# Attributes
+# Annotations
 
-Some types and methods can be annotated with attributes. The attribute list is fixed, but eventually (maybe) there will be user-defined attributes.
+Some types and methods can be annotated with built-in and user-defined annotations.  The built-in annotations include:
 
 ## Link
```

Comment:
Here should first be an overview list of built-in annotations with links to the respective sections.

Also, I think they should be moved to a child page. Except for `@[Flags]` they are not very commonly used by most developers. There is little value in exposing the specifics directly here.

It is much more important to describe how annotations are used in general and how you can define your own. That's what 90% of users are interested in.

So essentially copy the contents of `user_defined_annotations.md` here. And maybe add a paragraph at the beginning which focuses on explaining the general concept and application of annotations.